app/javascript/dashboard/mixins/time.js

```diff
@@ -1,16 +1,33 @@
-import fromUnixTime from 'date-fns/fromUnixTime';
 import format from 'date-fns/format';
-import formatDistanceToNow from 'date-fns/formatDistanceToNow';
+
+const divisions = [
+  { amount: 60, name: 'seconds' },
+  { amount: 60, name: 'minutes' },
+  { amount: 24, name: 'hours' },
+  { amount: 30, name: 'days' },
+  { amount: 12, name: 'months' },
+  { amount: Number.POSITIVE_INFINITY, name: 'years' },
+];
 
 export default {
   methods: {
-    messageStamp(time, dateFormat = 'h:mm a') {
-      const unixTime = fromUnixTime(time);
-      return format(unixTime, dateFormat);
+    messageStamp(unixTimestamp, dateFormat = 'h:mm a') {
+      return format(unixTimestamp * 1000, dateFormat);
     },
-    dynamicTime(time) {
-      const unixTime = fromUnixTime(time);
-      return formatDistanceToNow(unixTime, { addSuffix: true });
+    dynamicTime(unixTimestamp) {
```

Comment:
@KubesDavid Thanks for the PR. Can we rewrite this with `date-fns`? We are not aligned about adding more maintainable code into the repo.